specs/protocol/isthmus/derivation.md

```diff
@@ -16,11 +23,261 @@ The Isthumus hardfork activation block contains the following transactions, in t
 - L1 Attributes Transaction
 - User deposits from L1
 - Network Upgrade Transactions
+  - L1Block deployment
+  - GasPriceOracle deployment
+  - Operator Fee vault deployment
+  - Update L1Block Proxy ERC-1967 Implementation
+  - Update GasPriceOracle Proxy ERC-1967 Implementation
+  - Update Operator Fee vault Proxy ERC-1967 Implementation
+  - GasPriceOracle Enable Isthmus
   - EIP-2935 Contract Deployment
 
 To not modify or interrupt the system behavior around gas computation, this block will not include any sequenced
 transactions by setting `noTxPool: true`.
 
+## L1Block deployment
+
+The `L1Block` contract is upgraded to support the Isthmus operator fee feature.
+
+A deposit transaction is derived with the following attributes:
+
+- `from`: `0x4210000000000000000000000000000000000003`
+- `to`: `null`
+- `mint`: `0`
+- `value`: `0`
+- `gasLimit`: `425,000`
+- `data`: `0x60806040523480156100105...` ([full bytecode](../../static/bytecode/isthmus-l1-block-deployment.txt))
+- `sourceHash`: `0x3b2d0821ca2411ad5cd3595804d1213d15737188ae4cbd58aa19c821a6c211bf`,
+  computed with the "Upgrade-deposited" type, with `intent = "Isthmus: L1 Block Deployment"
+
+This results in the Isthmus L1Block contract being deployed to `0xFf256497D61dcd71a9e9Ff43967C13fdE1F72D12`, to verify:
+
+```bash
+cast compute-address --nonce=0 0x4210000000000000000000000000000000000003
+Computed Address: 0xFf256497D61dcd71a9e9Ff43967C13fdE1F72D12
+```
+
+Verify `sourceHash`:
+
+```bash
+cast keccak $(cast concat-hex 0x0000000000000000000000000000000000000000000000000000000000000002 $(cast keccak "Isthmus: L1 Block Deployment"))
+# 0x3b2d0821ca2411ad5cd3595804d1213d15737188ae4cbd58aa19c821a6c211bf
+```
+
+Verify `data`:
+
+```bash
+git checkout 905311d64d3df67649837ec57f0b0a4b6214ce31
```

Comment:
This is a copypasta - we don't use pnpm anymore. Please prove to me that the bytecode in the txt file is legit

Reply:
We are going to want to use a commit of a release. This means we will want to remove any rc suffix on the semver of the specific contracts being upgraded

Reply:
I updated to `make build-contracts `

Reply:
We should leave the exact commit and bytecode empty for now, we can merge the contracts in their own PR and then cut a release of them to know the exact bytecode